doc/usage/extensions/autodoc.rst

```diff
@@ -299,6 +299,15 @@ inserting them into the page source under a suitable :rst:dir:`py:module`,
 
         .. autoclass:: module.name::Noodle
 
+   * For objects defined in multiple modules, autodoc will generate both
+     specific directives and use the ``:canonical:`` option accordingly.
```

Comment:
It seems this describes the internal of autodoc itself. Does this explanation help users?